src/bundle/DependencyInjection/Compiler/SecurityLoginPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\DependencyInjection\Compiler;
+
+use EzSystems\EzPlatformAdminUi\Security\Authentication\RedirectToDashboardAuthenticationSuccessHandler;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Exception;
+
+class SecurityLoginPass implements CompilerPassInterface
+{
+    /**
+     * @param ContainerBuilder $container
+     *
+     * @throws Exception\InvalidArgumentException
+     * @throws Exception\ServiceNotFoundException
+     */
+    public function process(ContainerBuilder $container)
+    {
+        $successHandlerDef = $container->getDefinition('security.authentication.success_handler');
```

Comment:
Any reason we need a compiler pass instead of overriding a service in Yaml configuration?

Reply:
I also think overriding in yaml is more clear. Just put it in dedicated file i.e. `overrides.yml`.